src/Cafe/OS/libs/nsyshid/BackendLibusb.cpp

```diff
@@ -241,14 +252,31 @@ namespace nsyshid::backend::libusb
 						ret);
 			return nullptr;
 		}
+		std::vector<ConfigDescriptor> config_descriptors{};
+		for (uint8 i = 0; i < desc.bNumConfigurations; ++i)
+		{
+			auto [ret, config_descriptor] = MakeConfigDescriptor(dev, i);
+			if (ret != LIBUSB_SUCCESS || !config_descriptor)
+			{
+				cemuLog_log(LogType::Force, "Failed to make config descriptor {} for {:04x}:{:04x}: {}",
+							i, desc.idVendor, desc.idProduct, libusb_error_name(ret));
+			}
+			config_descriptors.emplace_back(std::move(config_descriptor));
```

Comment:
This puts invalid config descriptors in the `config_descriptors` vector.
I guess this might cause problems later on.